### DISCUSSION TITLE
V23 for new project

### DISCUSSION THREAD

**Discussion by jjones3535a** [pocketbase/pocketbase#5804] [Q&A]
I’ve been testing out v23 and it looks pretty good. This version seems to have a lot of refactoring and will be closer in the event pocket base gets to a v1.0. So I was hoping to use this version.

I’d like to use it for a small project and was hoping to host on pockethost. It seems they only host final versions. Any idea when version 23 will be final?

**Comment by chnfyi** [comment]
Last week the maintainer said in response to some issues that there is no estimate time of release yet.

He talked about like _near future_ so maybe before the end of the year.

It's a big release with breaking changes that require a lot of documentation updates too. It's huge so we have to be patient !
